### PR TITLE
Add footer widget and code copy features

### PR DESCRIPTION
**Pull request by rushenn**
Fixes: https://github.com/minio/gluegun/issues/60

### CODE REVIEW

File: lib/_sidebar.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+    <aside class="sidebar">
```
Comment (kaankabalak) [inline lib/_sidebar.erb]: This is the .erb that is generated by _sidebar-template.erb, I think we can remove this file from the commit

Comment (kaankabalak) [inline lib/_sidebar.erb]: This is the .erb that is generated from _sidebar-template.erb, I think we can remove this file from the commit.

**Comment by deekoder** [comment]
oh my prev pr removes the lib/_sidebar.erb.

File: site.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ Plugins:
       apiKey: "ALGOLIA_TOKEN"
       index: "INDEX_NAME"
   - Google Analytics: "UA-XXXXX-Y"
+  - Footer Widget:
```
Comment (deekoder) [inline site.yml]: We should call it slack.  Rushenn if this is not in the site.yml, there is no need to show the widget.

File: site.yml [hunk 3]
```diff
@@ -11,6 +11,9 @@ Plugins:
...
+      Label: "Talk to community"
+      Link: "/"
```
Comment (deekoder) [inline site.yml]: Do we give a link or a token for this integration?

File: site.yml (hunk 2 above)
Comment (kaankabalak) [reply site.yml]: I think people might also want to link the footer widget to other sites than Slack (e.g. adding a mailto link)

Comment (deekoder) [reply site.yml]: We could. But we could also be opinionated.

Comment (deekoder) [reply site.yml]: We don't have to block on this @rushenn

Comment (rushenn) [reply site.yml]: @deekoder 
I'll change the label to 'Slack'. This widget is conditional. Footer widget will only render if the parameters are set.

File: site.yml (hunk 3 above)
Comment (rushenn) [reply site.yml]: @deekoder It's a link.

**Comment by rushenn** [comment]
@kaankabalak Excluded _sidebar.erb from the commit. Please check.

**Pull request merged**